### PR TITLE
Configured ESLint + Lint fixes

### LINKED ISSUE #43: Implement linter

### PR DESCRIPTION
**Pull request by Adnan-Sait**
Added basic configuration to ESLint.
Added eslintignore file incase it is required in future.

Ran ESLint and fixed the code issues that were highlighted.

Closes #43

### CODE REVIEW

File: Modules/equipRandomEquipment.js [hunk 1]
```diff
@@ -40,23 +40,23 @@ export const equipRandomEquipment = (gearOwned, allGear) =>{
         'eyewear',
         'head',
         'headAccessory',
-      ].filter((t) => gearGroups.hasOwnProperty(t));
+      ].filter((t) => Object.prototype.hasOwnProperty.call(gearGroups, t));
```
Comment (ieahleen) [inline Modules/equipRandomEquipment.js:43]: can you change the eslint rules to have the other way valid?

File: Modules/startRandomQuest.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import { randomElementFromArray, fetchAPI, post } from "./utils.js";
 
+// eslint-disable-next-line no-unused-vars
```
Comment (ieahleen) [inline Modules/startRandomQuest.js:3]: what's unused here?

Comment (Adnan-Sait) [reply Modules/startRandomQuest.js:3]: In line 4, `userLevel` is unused in the function.
I did not want to change the function signature, so I added the eslint-disable

File: Modules/equipRandomEquipment.js (hunk 1 above)
Comment (Adnan-Sait) [reply Modules/equipRandomEquipment.js:43]: I'll look into this

Comment (Adnan-Sait) [reply Modules/equipRandomEquipment.js:43]: I can switch off this rule and revert these changes.
Shall I do that?

File: Modules/startRandomQuest.js (hunk 2 above)
Comment (ieahleen) [reply Modules/startRandomQuest.js:3]: You can remove it, it was needed for a previous thing

File: Modules/equipRandomEquipment.js (hunk 1 above)
Comment (ieahleen) [reply Modules/equipRandomEquipment.js:43]: Yes please :)

Comment (Adnan-Sait) [reply Modules/equipRandomEquipment.js]: I have made this change.

File: Modules/startRandomQuest.js (hunk 2 above)
Comment (Adnan-Sait) [reply Modules/startRandomQuest.js]: I have made this change

**Review by ieahleen — APPROVED** [review]

**Pull request merged**